src/LodeRunner.UI/src/services/configs.test.js

```diff
@@ -40,3 +40,34 @@ describe("checkConfigInputs", () => {
     }
   });
 });
+
+describe("addDefaultsToConfig", () => {
+  it("should replace all nulls with values", () => {
+    const config = { [CONFIG.id]: "123" };
+    addDefaultsToConfig(config);
+    Object.values(config).forEach(configValue => {
+      expect(configValue).not.toBeNull();
+    })
```

Comment:
What are your thoughts on adding an `expect(config.length).toEqual(Object.keys(CONFIG_OPTIONS).length);`  below line 50 just to make sure that it added key-value pairs from config_options